server/src/test/java/io/crate/integrationtests/RoleManagementIntegrationTest.java

```diff
@@ -292,4 +292,15 @@ public void test_create_user_jwt_properties_must_be_unique() {
             .hasHTTPError(CONFLICT, 4099)
             .hasMessageContaining("Another role with the same combination of jwt properties already exists");
     }
+
+    @Test
+    public void test_alter_user_jwt_properties() {
```

Comment:
Can't we avoid this itest and instead add a unit test in `TransportRoleActionTest`?